connector-slack/src/main/java/com/redhat/cloud/notifications/connector/slack/SlackCloudEventDataExtractor.java

```diff
@@ -13,8 +13,13 @@ public class SlackCloudEventDataExtractor extends CloudEventDataExtractor {
 
     @Override
     public void extract(Exchange exchange, JsonObject cloudEventData) {
+        System.out.println(cloudEventData);
         exchange.setProperty(TARGET_URL, cloudEventData.getString("webhookUrl"));
-        exchange.setProperty(CHANNEL, cloudEventData.getString("channel"));
-        exchange.getIn().setBody(cloudEventData.getString("message"));
+        JsonObject message = new JsonObject();
+        message.put("text", cloudEventData.getString("message"));
+        if (cloudEventData.getValue("channel") != null) {
+            message.put(CHANNEL, cloudEventData.getString("channel"));
```

Comment:
We could get rid of `ExchangeProperty` in this module since "channel" is no longer set on the exchange.
```suggestion
            message.put("channel", cloudEventData.getString("channel"));
```